tools/dotnet-scaffold-aspire/Commands/GetCommands.cs

```diff
@@ -33,7 +49,17 @@ public override int Execute(CommandContext context)
 
     public static class GetCmdsHelper
     {
-        internal static Parameter ProjectParameter = new() { Name = "--project", DisplayName = "Project Name", Description = "Project of choice for the scaffolding", Required = true, Type = BaseTypes.String, PickerType = InteractivePickerType.ProjectPicker };
-        internal static Parameter[] RedisParameters = [ProjectParameter];
+        internal static Parameter HostProjectParameter = new() { Name = "--host-project", DisplayName = "Aspire Host project file", Description = "Aspire host app for the scaffolding", Required = true, Type = BaseTypes.String, PickerType = InteractivePickerType.ProjectPicker };
+        internal static Parameter ApiProjectParameter = new() { Name = "--api-project", DisplayName = "API project file", Description = "API project associated with the Aspire Starter App", Required = true, Type = BaseTypes.String, PickerType = InteractivePickerType.ProjectPicker };
```

Comment:
I have questions/thoughts about these other parameter names but don't want to block the PR on them, we can discuss and change later.

Reply:
sounds good tyty